src/python/pants/notes/2.11.x.md

```diff
@@ -1,5 +1,61 @@
 # 2.11.x Release Series
 
+## 2.11.0.dev1 (Feb 22, 2022)
+
+### New Features
+
+* Support for `file`/`files` in JVM tests ([#14537](https://github.com/pantsbuild/pants/pull/14537))
+
+* Support complete_platforms for `python_awslambda`. ([#14532](https://github.com/pantsbuild/pants/pull/14532))
+
+* Support complete_platforms for Python GCFs. ([#14530](https://github.com/pantsbuild/pants/pull/14530))
+
+* Properly support filesystem specs in the `paths` goal. ([#14518](https://github.com/pantsbuild/pants/pull/14518))
+
+* Plumb Pex's `--complete-platform` for pex_binary. ([#14526](https://github.com/pantsbuild/pants/pull/14526))
+
+* Allow use of `parametrize` with the `jdk=` field of JVM targets ([#14510](https://github.com/pantsbuild/pants/pull/14510))
+
+* Adds the plugin names field to scala sources ([#14508](https://github.com/pantsbuild/pants/pull/14508))
+
+* Per-resolve scala plugins ([#14497](https://github.com/pantsbuild/pants/pull/14497))
+
+* jvm: teach dependency inference about multiple resolves ([#14491](https://github.com/pantsbuild/pants/pull/14491))
+
+* scala: add implicit dependency on scala-library ([#14450](https://github.com/pantsbuild/pants/pull/14450))
+
+* py_awslambda: pass through `include_requirements:bool` to `pex_binary` ([#14469](https://github.com/pantsbuild/pants/pull/14469))
```

Comment:
As a releaser, you can tweak these entries to be better. I'd recommend tweaking this:

> Add `include_requirements` field to `python_awslambda`